### PR TITLE
Propose team discussion policy

### PR DESCRIPTION
**Pull request by benjamingr**
This is just a proposal to bikeshed on for the collaborator team discussion board.

I have no strong feelings about having this particular policy - but I think that we should have a policy since currently it's unclear what form of information disclosure is appropriate.

### CODE REVIEW

File: Moderation-Policy.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ moderation request, please see [Requesting Moderation][]
   * [Temporary Interaction Limits][]
   * [Temporary and Indefinite Blocks][]
 * [Privacy of the nodejs/moderation Repository][]
+* [Privacy of the nodejs/collaborators Team Discussion Board][]
```
Comment (mhdawson) [inline Moderation-Policy.md]: When I use view the link does not seem to be quite right.

**Comment by Trott** [comment]
I'm not sure the Moderation Policy doc is the right place for this information. This isn't about Moderation Policy. Maybe this belongs in the MemberExpectations doc? That seems geared at "leaders" but whatever, the title says members... @nodejs/moderation for thoughts?

**Comment by Trott** [comment]
One reason I don't want it in the Moderation Policy is because I don't want @nodejs/moderation to be responsible for enforcement. Booting someone from the org for violating this seems more of a TSC and/or CommComm decision.

I'm also open to the idea that we don't really need this documented. The more things we have documented about how people are supposed to behave, the less likely they are to ever read it or remember it if they do read it. Is this important enough? (Not saying it is or isn't, but asking the question.)

**Comment by Tiriel** [comment]
I do agree with @Trott on some things.
I don't fully know the possibilities for enforcement of the moderation on GitHub, but completely removing someone from the org does not seem like a power that should befall the moderation team IMHO. More like at least a decision of the TSC or CommComm, and even preferably both.

Other than that, the base principle of enforcing basic privacy of some discussions seems ok to me.

**Comment by mhdawson** [comment]
I had the same question about removal, however, it mirrors this from the existing doc so the moderation team already has that power.:

```
The nodejs/moderation Repository is used to discuss the potentially sensitive details of any specific moderation issue. The repository is private but accessible to all Collaborators. The details of any issue discussed within the nodejs/moderation repository are expected to remain confidential and are not to be discussed in any public forum or social media service.

Any Collaborator found to be violating the privacy of the nodejs/moderation repository by repeatedly sharing or discussing the details of nodejs/moderation issues in any public forum or social media service risks being permanently removed from the Node.js GitHub organization.
```

**Comment by Trott** [comment]
> I had the same question about removal, however, it mirrors this from the existing doc so the moderation team already has that power.:

I can pretty much guarantee the Moderation team (at least as it is currently constituted) would never remove someone from the org without TSC and/or CommComm buy-in first. And event then, they might have TSC do it rather than Moderation team.

I don't think the discussion board contents necessitate the same degree of caution/privacy, but that's just my opinion. If others feel differently, cool.

I'd be in favor of modifying the text in the policy quoted by @mhdawson to make the removal **process** more clear. The challenge is to make sure it doesn't happen capriciously while also making sure it can happen quickly enough to be effective and not be bogged down by days or even weeks of process while a bad actor continues to act badly.

**Comment by Tiriel** [comment]
I'll second @Trott  on that.

The process is not defined at all, apart from the fact that the offender "risks" removal from the org. This kind of sanction is extreme and needs clarity IMO. And tbh I don't think it should befall the moderation team alone without TSC and/or CommComm approval.

(Although I do acknowledge that I don't think any member of the moderation would do it on their own, as was pointed before. but this sort of thing needs to be clearly stated.)

**Comment by benjamingr** [comment]
Added CC agenda and TSC agenda to get LGTMs and probably (hopefully) merge eventually or alter with feedback.

File: Moderation-Policy.md [hunk 2]
```diff
@@ -237,6 +238,22 @@ repository by repeatedly sharing or discussing the details of nodejs/moderation
...
+The nodejs/collaborators team discussion board is sometimes used for discussing 
+things that don't directly fit the nodejs/node repository. For example nominations
+of Contributors for Collaborator status might be discussed there in order to give
+existing Collaborators a chance to voice their opinion on the nomination.
+
+The nodejs/collaborators team discussion board is considered private but accessible to
+all Collaborators. The details of any issue discussed within it are expected to remain
+confidential and are not to be discussed in any public forum or social media service.
+
+Any Collaborator found to be violating the privacy of it by repeatedly sharing or
```
Comment (Trott) [inline Moderation-Policy.md]: `privacy of it` is a bit awkward here, although I see it mirrors `privacy of the nodejs/moderation repository` in the previous section. In both places, I would change `privacy` to `confidentiality`. And here, I would change `of it` to `of the nodejs/collaborators Team Discussion Board`.

**Comment by Trott** [comment]
> @Trott do we even want to land this?

@benjamingr If it's left up to me entirely, I'd say probably not. But I don't object if others want to land it. I thought you were advocating for it because you authored it.

**Comment by benjamingr** [comment]
@Trott well, I feel like we should have _a_ policy that's explicit but I don't have a strong feeling about what policy it should be as long as we're consistent in enforcing it. 

I tagged this as tsc/cc agenda so that (hopefully) one of those bodies feels strongly enough about such a policy to propose one (or accept this one)

**Comment by MylesBorins** [comment]
Perhaps we should focus this on what topics should be being brought to the collaborators board.

We should be pushing for transparency whenever possible

**Comment by refack** [comment]
Following the discussion in the TSC meeting I would like to propose adding the opposite statement:

```md
## Transparency of the nodejs/collaborators Team Discussion Board
 
The nodejs/collaborators team discussion board is sometimes used for discussing 
things that don't directly fit the nodejs/node repository. The nodejs/collaborators team discussion board should not be considered private. The details of any issue discussed within it might be made public, and ATM are only limited to org members due to GitHub's implementation of this feature.
```

I'm not sure where this paragraph should go. I would suggest somewhere closer to the `Start a discussion with @nodejs/collaborators` button...

**Comment by Trott** [comment]
(Since this was discussed at the TSC meeting today, should the tsc-agenda label be removed? Same question for all other issues discussed today. Let's not have them pop up on the agenda next week unless they're supposed to!)

**Comment by benjamingr** [comment]
@Trott what was the resolution in the TSC meeting?

**Comment by mhdawson** [comment]
In respect to:

```I tagged this as tsc/cc agenda so that (hopefully) one of those bodies feels strongly enough about such a policy to propose one (or accept this one)```

There was nobody who felt strongly enough to take either of the actions you mentioned.

**Comment by benjamingr** [comment]
Ok, let's keep CC agenda then and remove the TSC agenda tag, maybe the CC can weigh in on this (cc @nodejs/community-committee )

**Comment by refack** [comment]
The CC has decided to recommend that in general "GitHub's" team discussions should be made as public as possible, and considered so.
Chartered sub groups could decide on different rules and enforce on their own.

I'm removing the cc-agenda tag and closing this in favor of https://github.com/nodejs/admin/pull/161

**Pull request closed without merging**